vagrant/boxes.d/99-local.yaml.example

```diff
@@ -1,4 +1,20 @@
 ---
+centos9-katello-devel:
+  primary: true
+  box: centos9-stream
+  ansible:
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
+    variables:
+      ssh_forward_agent: true
+      foreman_devel_github_push_ssh: True
+      katello_devel_github_username: <REPLACE ME>
+      foreman_repositories_environment: staging
+      katello_repositories_environment: staging
+      foreman_installer_options:
+        - "--foreman-proxy-content-enable-ostree=true"
+        - "--katello-devel-modulestream-nodejs=18"
+
 centos8-katello-devel:
```

Comment:
I think we should delete this box, or move it to Alma8, as in the current state it won't work anymore.

Reply:
I moved it to Alma.